docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md

```diff
@@ -212,3 +244,7 @@ Once Excel starts, you can use the task pane buttons to store or get shared data
 
```

Comment:
Small nit, but in the last step we're asked to call `npm run start`. Two things:
- We called `npm start` earlier. Should this be the same command?
- The tutorial never has the user close the add-in. Should we have them close and reopen? Or should we remove the earlier start? FWIW, I never closed the previous run and didn't get the new functions. Closing and rerunning npm start refreshed things.